tests/datasets/test_advance.py

```diff
@@ -15,6 +14,7 @@
 
 import torchgeo.datasets.utils
 from torchgeo.datasets import ADVANCE
+from torchgeo.datasets.utils import Path
```

Comment:
This is incorrect, the correct type is `pathlib.Path`

Reply:
Have replaced it with Path , so that it stays consistent with the codebase.

Reply:
`torchgeo.datasets.utils.Path` and `pathlib.Path` are two different things. The latter is a subset of the former. We should be as specific as possible.

Reply:
@adamjstewart ok will revert these changes.